compiler/env/OMRClassEnv.hpp

```diff
@@ -82,6 +82,7 @@ class OMR_EXTENSIBLE ClassEnv
 
    bool isAbstractClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazzPointer) { return false; }
    bool isInterfaceClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazzPointer) { return false; }
+   bool isConcreteClass(TR::Compilation *comp, TR_OpaqueClassBlock *clazzPointer) { return false; }
```

Comment:
Since `isConcrete` means it's not abstract and it's not interface and since those last two queries return false by default, I would make `isConcrete` return true by default.